tests/src/rules/newline-after-import.js

```diff
@@ -7,9 +7,9 @@ const ruleTester = new RuleTester()
 
 ruleTester.run('newline-after-import', require('rules/newline-after-import'), {
   valid: [
-    "var path = require('path');\nvar foo = require('foo');\n",
-    "require('foo');",
-    "switch ('foo') { case 'bar': require('baz'); }",
+    `var path = require('path');\nvar foo = require('foo');\n`,
```

Comment:
I don't mind changing this to backticks, but then we might as well replace the `\n` by in-code line breaks.
(Not necessary for this PR, but if you feel like changing that at some point later, that's would be nice :D).